stdlib/Distributed/src/cluster.jl

```diff
@@ -666,16 +698,16 @@ function create_worker(manager, wconfig)
     @async manage(w.manager, w.id, w.config, :register)
     # wait for rr_ntfy_join with timeout
     timedout = false
-    @async (sleep($timeout); timedout = true; put!(rr_ntfy_join, 1))
+    @async (sleep(maxwait); timedout = true; put!(rr_ntfy_join, 1))
```

Comment:
Keep interpolation of value?

Reply:
That doesn't seem necessary, maxwait does not change.

Reply:
now removed in https://github.com/JuliaLang/julia/pull/32290/commits/c7ceaea79ab73a9e1f6db7bd7f2dcbada4c4787c